jenkins-deploy

```diff
@@ -1,18 +1,13 @@
 #!/bin/sh
 set -eu
 
-DIR=$1
-ENV=$2
-
-cd dpe-$DIR
+ENV=$1
```

Comment:
I don't think we should specify an environment since we only deploy to test with this job?